build/artifacts.props

```diff
@@ -46,6 +46,7 @@
     <PackageArtifact Include="Microsoft.AspNetCore.Authentication.OAuth" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Authentication.Twitter" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
+    <PackageArtifact Include="Microsoft.AspNetCore.Authentication.WsFederation" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
```

Comment:
@DamianEdwards - just checking: this brings in some additional Microsoft-built dependencies, such as Microsoft.IdentityModel.Protocols.WsFederation and others. Is that OK for the .App package?

Reply:
They're largely the same dependencies as OpenIdConnect.

Reply:
Ah ok then I'm not concerned. Thanks!